deploy/charts/rook-ceph-cluster/templates/cephecblockpool.yaml

```
@@ -43,4 +46,4 @@ parameters:
 #mounter: rbd-nbd
 allowVolumeExpansion: {{ $cephEcStorage.allowVolumeExpansion }}
 reclaimPolicy: {{ $cephEcStorage.reclaimPolicy }}
-{{ end }}
+{{ end }}
```

```suggestion
{{ end }}

```
can you have empty extra line in the end?